app/grants/models.py

```diff
@@ -1691,6 +1691,16 @@ class Contribution(SuperModel):
     )
     anonymous = models.BooleanField(default=False, help_text=_('Whether users can view the profile for this project or not'))
 
+    @property
+    def blockexplorer_url(self):
+        if self.checkout_type == 'eth_zksync':
+            return f'https://zkscan.io/explorer/transactions/{self.split_tx_id.replace("sync-tx:", "")}'
+        if self.checkout_type == 'eth_std':
+            network_sub = f"{{self.subscription.network}}." if self.subscription and self.subscription.network != 'mainnet' else ''
+            return f'https://{network_sub}etherscan.io/tx/{self.split_tx_id}'
+        # TODO: support all block explorers for diff chains
```

Comment:
@chibie we'll have to keep a track of this to update next week with all the chains